site/docs/install-compile-source.md

```diff
@@ -36,39 +40,18 @@ For example on Ubuntu Linux you can install these requirements using the followi
 sudo apt-get install build-essential openjdk-8-jdk python zip unzip
 ```
 
-#### Windows
+### 2.  Download and unpack Bazel's source files (distribution archive)<a name="download-dist"></a>
 
-Ensure you have installed:
-
-*   [MSYS2 shell](https://msys2.github.io/)
-
-*   **The required MSYS2 packages.** Run the following command in the MSYS2
-    shell:
-
-    ```
-    pacman -Syu zip unzip
-    ```
-
-*   **The Visual C++ compiler.** Install the Visual C++ compiler either as part
-    of Visual Studio 2015 or newer, or by installing the latest [Build Tools
-    for Visual Studio 2017](https://aka.ms/BuildTools).
-
-*   **JDK 8.** You must install version 8 of the JDK. Versions other than 8 are
-    *not* supported.
-
-*   **Python**. Versions 2 and 3 are supported. You need the
-    Windows-native version (downloadable from [https://www.python.org](https://www.python.org)).
-    Versions installed via pacman in MSYS2 will not work.
-
-### 2.  Download and unpack Bazel's source files (distribution archive)
+(This is the same for Windows.)
```

Comment:
Can we move this section before?
You could reduce the number of jumps by using this structure:


1. Download and unpack Bazel's source files
2. Bootstrapping Bazel on Ubuntu Linux, macOS, and other Unix-like systems
    * Install the prerequisites
    * Bootstrap Bazel
3. Bootstrapping Bazel on Windows
    * Install the prerequisites
    * Bootstrap Bazel
4. Development snapshots of Bazel

Reply:
Good idea! Done. PTAL.